### PR TITLE
WIP - Proposed 1.7.0 roadmap

### PR DESCRIPTION
**Pull request by dsu-igeek**
Signed-off-by: Dave Smith-Uchida <dsmithuchida@vmware.com>

Proposed 1.7.0 roadmap for review

# Please indicate you've done the following:

- [X] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [X] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [X] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -15,23 +15,55 @@
...
+
+
+##### Desired items
+|Issue|Description|
+|---|---|
+|[3533](https://github.com/vmware-tanzu/velero/issues/3533)|Upload Progress Monitoring|
+|[2922](https://github.com/vmware-tanzu/velero/issues/2922)|Plugin timeouts|
+|[3500](https://github.com/vmware-tanzu/velero/issues/3500)|Use distroless containers as a base|
+|[3535](https://github.com/vmware-tanzu/velero/issues/3535)|Design doc for multiple cluster support|
+|[3536]|https://github.com/vmware-tanzu/velero/issues/3536|
```
Comment (carlisia) [inline ROADMAP.md:44]: Missing parens for the link https://github.com/vmware-tanzu/velero/issues/3536

**Review by carlisia — CHANGES_REQUESTED** [review]
Thanks for doing this, looks good to me. Seems of a good size for a release.


I pointed out missing parens is all.

File: ROADMAP.md (hunk 1 above)
Comment (dsu-igeek) [reply ROADMAP.md]: Thanks, that should be fixed now.

**Review by nrb — APPROVED** [review]
Thanks for providing a break down of these items with links - that was on my todo list for a long time.

**Review by carlisia — APPROVED** [review]

**Pull request merged**